cli_client/python/timesketch_cli_client/commands/sketch.py

```diff
@@ -30,14 +30,73 @@ def list_sketches(ctx):
         click.echo(f"{sketch.id} {sketch.name}")
 
 
-@sketch_group.command("describe")
+@sketch_group.command(
+    "describe", help="Describe the active sketch. Attributes are not included."
+)
 @click.pass_context
 def describe_sketch(ctx):
     """Show info about the active sketch."""
     sketch = ctx.obj.sketch
-    # TODO (berggren): Add more details to the output.
+    output = ctx.obj.output_format
+
+    if output == "json":
+        click.echo(sketch.__dict__)
+        return
     click.echo(f"Name: {sketch.name}")
     click.echo(f"Description: {sketch.description}")
+    click.echo(f"Status: {sketch.status}")
+
+
+@sketch_group.command("attributes")
+@click.pass_context
+def list_attributes(ctx):
+    """List all attributes."""
+    sketch = ctx.obj.sketch
+
+    attributes = sketch.attributes
+    click.echo(attributes)
+
+
+@sketch_group.command("remove_attribute")
+@click.option("--name", required=True, help="Name of the attribute.")
+@click.option("--ontology", required=True, help="Ontology of the attribute.")
+@click.pass_context
+def remove_attribute(ctx, name, ontology):
+    """Remove an attribute from a sketch.
+
+    Args:
+        name: Name of the attribute.
+        ontology: Ontology of the attribute.
+    """
+    sketch = ctx.obj.sketch
+    if sketch.remove_attribute(name, ontology):
+        click.echo(f"Attribute removed: {name} {ontology}")
```

Comment:
Would it be better (for this and other feedback regarding attributes) to epxlicit which is the name and which is the ontology?

```
name: {name}
ontology: {ontology}
value: {value}
```

Reply:
yes